recommender/engine/rl_agent/models/actor_model.py

```diff
@@ -1,37 +1,81 @@
+# pylint: disable=missing-module-docstring, invalid-name, too-many-arguments, no-member
+
+from typing import Tuple
+from itertools import chain
+
 import torch
+from torch import nn
+
+from recommender.engine.rl_agent.models.history_model import HistoryModel
+from recommender.engine.rl_agent.models.search_phrase_model import SearchPhraseModel
 
 
-class ActorModel(torch.nn.Module):
-    def __init__(self, K: int, SE: int, history_embedder: torch.nn.Module):
+class ActorModel(nn.Module):
```

Comment:
Please add:
- `ACTOR_1 = "actor model 1"`
- `ACTOR_2 = "actor model 2"`
constants at the top of the actor_model.py module. They will be used as names for saving and loading appropriate actor models in the RL Agent Recommender